swiftformat/Dockerfile

```
@@ -0,0 +1,34 @@
+# ================================
+# Build image
+# ================================
+FROM swift:latest as build
+
+LABEL maintainer="JacobCXDev <jacobcxdev@gmail.com>"
+
+# Setup a build area
+WORKDIR /build
+
+# Clone and build SwiftFormat
+RUN git clone https://github.com/nicklockwood/SwiftFormat .
```

Can we do something like this, to make the version we're building explicit?

```suggestion
ENV SWIFT_VERSION 0.49.3
RUN git clone https://github.com/nicklockwood/SwiftFormat .
RUN git checkout "$SWIFT_VERSION"
```

There's probably an efficient way to shallow-clone the tag only, but I'd have to figure that out.

---

> I'm going to request that you add some tests before I merge this. It's important to make sure things are (and continue to be) functional given that I'm not usually familiar with all the auto-formatters present as Restylers. Any other Restyler's `info.yaml` can be used as an example.

I was planning on adding tests, but the formatter is configurable via the existence of a `.swiftformat` file (see https://github.com/nicklockwood/SwiftFormat#config-file); might this not interfere with any hard-coded tests, or am I misunderstanding how they work?